spec/controllers/reviews_controller_spec.rb

```diff
@@ -0,0 +1,129 @@
+require 'rails_helper'
+
+RSpec.describe ReviewsController, type: :controller do
```

Comment:
in this test I'm missing some validation test - The review should have at least 3 char length or, rating value is between 0-5

Reply:
I did the columns validations at the 'Review.rb' file. Isn't it enough for covering these tests?

Reply:
you can add tests to it (not mandatory)